### PR TITLE
refactored admin to be more structured

### PR DESCRIPTION
**Pull request by czpython**
Moved all admin classes and forms to an ``admin`` folder.
This is to separate admin specific forms and views

### CODE REVIEW

File: aldryn_forms/admin/formdata.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+class FormDataAdmin(BaseFormSubmissionAdmin):
+    change_list_template = 'admin/aldryn_forms/formsubmission/change_list.html'
+    export_form = FormDataExportForm
+    readonly_fields = [
+        'name',
+        'data',
+        'language',
+        'sent_at',
+        'get_recipients_for_display'
```
Comment (intellectronica) [inline aldryn_forms/admin/formdata.py]: trailing comma, just because it's customary

File: aldryn_forms/admin/forms.py [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+from django.conf import settings
+from django.contrib.admin.widgets import AdminDateWidget
+from django.utils import timezone
+from django.utils.text import slugify
+from django.utils.translation import ugettext_lazy as _
+
+from ..models import FormData, FormSubmission
+
+
+def form_choices(modelClass):
```
Comment (intellectronica) [inline aldryn_forms/admin/forms.py]: where is `modelClass` defined?

File: aldryn_forms/admin/forms.py [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+        self.fields['form_name'].choices = form_choices(modelClass=self.model)
+
+    def clean(self):
+        if self.errors:
+            return self.cleaned_data
+
+        queryset = self.get_queryset()
+
+        if queryset.count() >= self.excel_limit:
+            error_message = _("Export failed! More then 65'536 entries found, exceeded Excel limitation!")
```
Comment (mkoistinen) [inline aldryn_forms/admin/forms.py]: Since we're using EN as the base language, should be "...65,536...".

**Comment by mkoistinen** [comment]
LGTM

File: aldryn_forms/admin/forms.py (hunk 3 above)
Comment (intellectronica) [inline aldryn_forms/admin/forms.py]: Loooooooooooong

**Comment by intellectronica** [comment]
:+1:

**Pull request merged**